packages/block-editor/src/components/block-list/content.scss

```diff
@@ -422,7 +422,7 @@ _::-webkit-full-page-media, _:future, :root .has-multi-selection .block-editor-b
 	}
 }
 
-.block-editor-block-list__block .block-editor-block-list__block-html-textarea {
+.block-editor-block-list__block .block-editor-block-list__block-html-textarea textarea {
```

Comment:
This is necessary because `TextareaControl` has a couple of wrappers around the `textarea` element.